Makefile

```diff
@@ -64,15 +68,15 @@ verify: tidy generate format lint ## Verify the current code generation and lint
 
 VERSION_FLAGS=-ldflags "-X $(VERSION_PATH).GitCommit=$(GIT_COMMIT) -X $(VERSION_PATH).ComboVersion=$(COMBO_VERSION) -X $(VERSION_PATH).KubernetesVersion=$(KUBERNETES_VERSION)"
 build-cli: ## Build the CLI binary. Specify VERSION_PATH, GIT_COMMIT, or KUBERNETES_VERSION to change the binary version. You may also specify BUILD_OS and BUILD_ARCH to change the build's binary. 
-	$(Q) CGO_ENABLED=0 GOOS=$(BUILD_OS) GOARCH=$(BUILD_ARCH) go build $(VERSION_FLAGS) -o ./bin/combo
+	$(Q) CGO_ENABLED=0 GOOS=$(BUILD_OS) GOARCH=$(BUILD_ARCH) go build $(VERSION_FLAGS) -o ./combo
```

Comment:
nit: I think the convention is to put the binary under `/bin` -- is there a reason to put it at the top level?

Reply:
I ran into some issues with the different binary locations between local/CI builds and ended up copying what is in RukPak. The final container has the binary under `/bin`.